src/courses/mappers/11.md

```diff
@@ -0,0 +1,54 @@
+---
+order: 11
+next: 12.md
+title: Mapper Preparations - Setting Up
+author: Charles Hu
+---
+
+## Environment Set Up
+
+Node.js (a runtime environment for JavaScript/TypeScript) and Yarn (a package manager for JavaScript/TypeScript) are external utilities which are utilized extensively within this guide. The following section details their installation process.
+
+Linux/Mac OS:
+
+1. Install [nvm](https://github.com/nvm-sh/nvm#install--update-script).
+  
+  - 1a. Use either of the following commands to install nvm:
+    ```shell
+    curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+    ```
+    ```shell
+    wget -qO- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+    ```
+  
+  - 1b. Either restart the terminal or run the following commands to use nvm:
+    ```shell
+    export NVM_DIR="$([ -z "${XDG_CONFIG_HOME-}" ] && printf %s "${HOME}/.nvm" || printf %s "${XDG_CONFIG_HOME}/nvm")"
+    [ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh" # This loads nvm
+    ```
+
+2. Run the following command to install and use Node.js v18:
+```shell
+nvm install 18
+```
+
+3. Install [Yarn](https://yarnpkg.com/getting-started/install)
```

Comment:
add some subbullets here too, ex. to say to do a global install of yarn via `npm install -g yarn`

Reply:
note: yarn officially uses corepack for installs instead of npm

Reply:
TIL - npm is how we recommend folks install yarn on the heimdall docs tho.  also does that work with yarn 1.x or is it only on yarn 2 (berry?)

honestly with all the improvements that have been happening in the space, i'm of the opinion we should spend some time to figure out if we should move to the latest version of yarn, or to one of the alternatives that are out there.

Reply:
Is there any reason for why we are using Yarn to begin with?